### PR TITLE
Feature/calculator

### PR DESCRIPTION
**Pull request by mateugazi**

### CODE REVIEW

File: src/app/calculate.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+async function calculate(sourceAmount, sourceUnit, targetUnit, apiKey) {
+    if (isNaN(sourceAmount)) {
+        return 'Please enter the correct number';
+    } else {
+        const response = await fetch(`https://api.spoonacular.com/recipes/convert?ingredientName=this&sourceAmount=${sourceAmount}&sourceUnit=${sourceUnit}&targetUnit=${targetUnit}&apiKey=${apiKey}`)
```
Comment (mateugazi) [inline src/app/calculate.js:5]: I check if the fetch return error later

File: src/app/calculator.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { MainMenu } from './MainMenu.js';
+
+MainMenu()
```
Comment (kingam280) [inline src/app/calculator.js:3]: You should import MENU from GlobalData and add active page here

File: calculator.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                    <option value="kilogram" class="calculator__form__to__option">Kilogram</option>
+                </select>
+                <button type="button" class="calculator__form__calculate">Calculate</button>
+                <button type="button" class="calculator__form__clear">Clear the output</button>
+                <div class="calculator__form__output">Answer goes here</div>
+            </form>
+        </div>
+    </div>
+
+    <script src="src/index.js" type="module"></script>
```
Comment (kingam280) [inline calculator.html:57]: do you need it when you have your own file?

File: test/calculator.test.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+//jest.mock('../calculate')
+
+const { calculate } = require('../src/app/calculate');
+const apiKey = '7c1cea1c1c664db5a38edbf2dd21484e';
+
+test('xd', () => {
```
Comment (kingam280) [inline test/calculator.test.js:6]: 'xd'? :D

File: calculator.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Calculator</title>
+    <link rel="stylesheet" href="styles/App.css" />
```
Comment (kingam280) [inline calculator.html:9]: isn't it better to upload your css file? it may help avoid potential conflicts

**Review by kingam280 — CHANGES_REQUESTED** [review]
You don't have all changes from main. For example menu doesn't look like it should

**Comment by mateugazi** [comment]
I changed everything you requested except file with 'xd'

**Review by kingam280 — APPROVED** [review]

**Review by Yoruko90 — APPROVED** [review]

**Review by Yoruko90 — APPROVED** [review]

**Review by neacatsu — APPROVED** [review]

File: src/app/calculate.js (hunk 1 above)
Comment (MaDud) [reply src/app/calculate.js:5]: Not complain just remaind :)

**Review by MaDud — APPROVED** [review]
It looks fine for me. Good job

File: test/calculator.test.js (hunk 4 above)
Comment (radekwojpl2) [reply test/calculator.test.js:6]: Better xd then something else XD

**Review by radekwojpl2 — APPROVED** [review]

**Review by JotGabiga — APPROVED** [review]

**Pull request merged**